hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -647,12 +647,10 @@ public Handler getHandler(ContainerProtos.ContainerType containerType) {
 
   @Override
   public void setClusterId(String clusterId) {
-    Preconditions.checkNotNull(clusterId, "clusterId Cannot be null");
-    if (this.clusterId == null) {
-      this.clusterId = clusterId;
-      for (Map.Entry<ContainerType, Handler> handlerMap : handlers.entrySet()) {
-        handlerMap.getValue().setClusterID(clusterId);
-      }
+    Preconditions.checkNotNull(clusterId, "clusterId cannot be null");
```

Comment:
@jianghuazhu,  could you also revert this change if there is no specific reason?  The general review guideline that I followed, is focus on the problem, and try not to touch irrelevant code as mush as possible.

Reply:
Thanks @ChenSammi .
I modified some code here. There are some main reasons:
1. The errorMessage contained in Preconditions.checkNotNull() has a typo.
`Cannot` should be changed to `cannot`.
2. The old code added an if check, which means that clusterId can only be assigned once, which is too strict.
```
if (this.clusterId == null) {
  this.clusterId = clusterId;
  ...... 
}   
```
These are related to clusterId, so I updated them together.
This is my idea.
What do you think?

Reply:
I see. Let's keep the typo change. 
It's expected that only one clusterId is valid during the DN lifetime, that clusterId is returned from SCM after DN registered to the SCM.  So I will prefer keep the clusterId null check here.

Reply:
Thanks @ChenSammi .
I have updated it.